io.embrace.sdk/Samples/Demo/Scripts/SetupEmbraceDemo.cs

```diff
@@ -21,6 +21,7 @@ void Start()
             #if DeveloperMode && UNITY_IOS
             // This setup is for Embrace Developer Mode on iOS only.
             Embrace.Instance.StartSDK(new EmbraceStartupArgs(AppId, 
+                EmbraceConfig.Default,
```

Comment:
just confirming that for customers this is a non-breaking change, e.g. the call they were previously making into the Unity API to start the SDK would continue to work without passing in the additional config argument?

Reply:
That will be the case for any customers not using the `#DeveloperMode` define. Which is something we only maintain for internal use anyways so they shouldn't be.

This is because prior the `EmbraceStartupArgs` would read: `AppId, Url, Url, Url, Url` in DeveloperMode.
But now it'll read `AppId, Default, Url, Url, Url, Url`